### PR TITLE
Update camera properties and fix test suite

### PR DESCRIPTION
**Pull request by AnthonyHorton**
Update temperature control related camera property names in the Pyro camera classes to match those now used in POCS camera classes, and fix all the broken tests.

### CODE REVIEW

**Review by wtgee — APPROVED** [review]
:+1: Sorry I missed these yesterday. They weren't failing because the tests are just designed to skip them upon fail. Could probably clean that up in some fashion but more work than it is worth.

**Comment by AnthonyHorton** [comment]
>  Sorry I missed these yesterday. They weren't failing because the tests are just designed to skip them upon fail. Could probably clean that up in some fashion but more work than it is worth.

https://github.com/panoptes/POCS/pull/909 enables those tests to be run, by using the SDK camera simulator instead of the DSLR one.

**Comment by AnthonyHorton** [comment]
Tests are now only failing because it's trying to use the SDK camera simulators, which won't work until https://github.com/panoptes/POCS/pull/909 gets merged. I have the test suite running successfully locally.

File: huntsman/camera/pyro.py [hunk 1]
```diff
@@ -450,30 +450,30 @@ def readout_time(self):
...
-        temperature = self._camera.ccd_set_point
+    def target_temperature(self):
+        temperature = self._camera.target_temperature
         return temperature.to(u.Celsius).value
 
-    @ccd_set_point.setter
-    def ccd_set_point(self, set_point):
-        self._camera.ccd_set_point = set_point
+    @target_temperature.setter
+    def target_temperature(self, set_point):
```
Comment (wtgee) [inline huntsman/camera/pyro.py]: Not a big deal, but for consistency.

```suggestion
    def target_temperature(self, target):
```

File: huntsman/camera/pyro.py [hunk 2]
```diff
@@ -450,30 +450,30 @@ def readout_time(self):
...
+        self._camera.target_temperature = set_point
```
Comment (wtgee) [inline huntsman/camera/pyro.py]:
```suggestion
        self._camera.target_temperature = target
```

**Review by wtgee — APPROVED** [review]
LGTM.

Unfortunately some of this would also change again when/if docker becomes default.

**Pull request merged**